myrights.go

```diff
@@ -0,0 +1,7 @@
+package imap
+
+// MyrightsData is the data returned by the MYRIGHTS command.
+type MyrightsData struct {
+	Mailbox string
+	Rights  string
```

Comment:
Can we use a better type instead of `string`?

Maybe something among the lines of:

```go
type RightSet string

type Right byte

const (
    RightLookup = Right('l')
    …
)
```

Reply:
fixed